Yoast/wordpress-seo
# Moves SEMrush PHP tests to the new directory and added missing tests for options action

## Context


* Moves SEMrush PHP tests to the new unit test directory and added missing tests for options action.

## Summary


This PR can be summarized in the following changelog entry:

* Moves SEMrush PHP tests to the new unit test directory and added missing tests for options action.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch and ensure you've ran `composer install`
* Run the unit tests and ensure that they all pass.

## UI changes

## Documentation

## Quality assurance


Fixes #
